add autoTestDiscoverOnSavePattern to python testing (#8070)
microsoft/vscode-docs: docs/python/testing.md

By default, the Python extension attempts to discover tests once you enable a framework. You can also trigger test discovery at any time using the **Test: Refresh Tests** command from the Command Palette.

> `python.testing.autoTestDiscoverOnSaveEnabled` is set to `true` by default, meaning that test discovery is also performed automatically whenever you add, delete, or update any Python file in the workspace. To disable this feature, set the value to `false`, which can be done either in the Settings editor or in the `settings.json` file as described in the VS Code [Settings](/docs/editor/settings.md) documentation. You will need to reload the window for this setting to take effect. If `python.testing.autoTestDiscoverOnSaveEnabled` is set to true, you can refine when auto test discovery occurs by using the `python.testing.autoTestDiscoverOnSavePattern` setting, which specifies a glob pattern that determines which file changes trigger test discovery. Its default value is `**/*.py`.

Reviewer: - Suggest to move the sentence about reloading the window to the end of the paragraph and shorten the text a bit:

```suggestion
`python.testing.autoTestDiscoverOnSaveEnabled` is set to `true` by default, meaning that test discovery is also performed automatically whenever you add, delete, or update any Python file in the workspace. To disable this feature, set the value to `false`. You can refine for which files auto test discovery occurs by specifying a glob pattern in the `python.testing.autoTestDiscoverOnSavePattern` setting. Its default value is `**/*.py`.

You can [configure the settings](/docs/editor/settings.md) in the Settings editor or directly in the `settings.json` file. You need to reload the window for the test discovery setting to take effect.
```

| Setting<br/>(python.testing.) | Default | Description |
| --- | --- | --- |
| autoTestDiscoverOnSaveEnabled | `true` | Specifies whether to enable or disable auto run test discovery when saving a test file. You may need to reload the window after making changes to this setting for it to be applied. |

> | autoTestDiscoverOnSavePattern | `**/*.py` | Specifies a glob pattern so  that determines which file changes trigger auto test discovery when `autoTestDiscoverOnSaveEnabled` is true. |

Reviewer: ```suggestion
| autoTestDiscoverOnSavePattern | `**/*.py` | Specifies a glob pattern that determines which file changes trigger auto test discovery when `autoTestDiscoverOnSaveEnabled` is `true`. |
```
